### PR TITLE
Multiple Improvements

### PR DESCRIPTION
**Pull request by gtomitsuka**
This commit:
- Adds a README.md for people who don't know what this repo is about
- Beautifies the HTML file
- Changes the google.co.uk link to google.com; It will redirect to your local Google website.
- Adds proper dashes in link so that it won't need to redirect(commit [df7fe55](https://github.com/jackmawer/Teenage-Programmers-Website/commit/df7fe55367e300fc5d44c4d80a3abc103edb788e))
- Adds the `type="text/css"` so that the W3 Validator passes.
- Removes the unnecessary test.html.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,91 +1,167 @@
...
+      <h2>
+        Rules
+      </h2>
+      <h3>
+        The rules for the room are simple:
+      </h3>
+      <ul>
+        <li>
+          Before asking a question here, ask it on 
+          <a TARGET="_top" href="http://stackoverflow.com">
```
Comment (thomas-daniels) [inline index.html:105]: What's the purpose of target="_top" here?

**Comment by gtomitsuka** [comment]
It was always there. Good catch! Will remove soon.

**Comment by gtomitsuka** [comment]
Oops :P

**Comment by jackmawer** [comment]
Pull request declined.
Reason: TARGET="_top" is required for the page to function right now!

**Comment by gtomitsuka** [comment]
Okay. Reverting the last commit.

**Comment by thomas-daniels** [comment]
@jackmawer Why is it?

**Comment by gtomitsuka** [comment]
Commit reverted.

**Comment by jackmawer** [comment]
Pull request accepted.

**Pull request merged**